### PR TITLE
docs(rxButton): Improve usage documentation.

### PR DESCRIPTION
**Pull request by megalord**
This is a pretty old component that has almost no documentation.
screen shot 2015-07-08 at 1 02 42 pm

### CODE REVIEW

File: src/rxButton/README.md [hunk 1]
```diff
@@ -2,8 +2,16 @@
...
 [API Documentation](ngdocs/index.html#/api/rxButton.directive:rxButton)
 
-Provides styling for various types of buttons. See the .less file for examples.
+An **element directive** used to create buttons with a dynamically-displayed loading indicator. This is used as a replacement for `<button>` elements in scenarios where the button has multiple states.
 
-## rxButton directive
+## Usage
 
-This directive is used to allow dynamic showing/hiding of a spinner/loading indicator.
+The state of the button is controlled via the `toggle` attribute, which disables the button and replaces the `default-msg` with the `toggle-msg` as the button's text.  There are no defaults for these messages, so they must be defined if the toggle behaviour is desired.  While the button is in the toggled state, it is also disabled (no matter what the value of `disable` is.
```
Comment (parlarjb) [inline src/rxButton/README.md]: I like that you spelled `behaviour` the correct way, but we should use the silly American spelling for consistency with the rest of the documentation.

**Comment by parlarjb** [comment]
LGTM

File: src/rxButton/README.md (hunk 1 above)
Comment (megalord) [inline src/rxButton/README.md]: I hoped you would find this :)

Comment (parlarjb) [inline src/rxButton/README.md]: Roughly 17% of the Canadian identity is in defining how we are different from Americans, this sort of thing always jumps out at me :)

**Pull request merged**